api/v1beta1/cloudstackmachine_webhook.go

```diff
@@ -82,6 +82,11 @@ func (r *CloudStackMachine) ValidateUpdate(old runtime.Object) error {
 	oldSpec := oldMachine.Spec
 
 	errorList = webhookutil.EnsureBothFieldsAreEqual(r.Spec.Offering.ID, r.Spec.Offering.Name, oldSpec.Offering.ID, oldSpec.Offering.Name, "offering", errorList)
+	errorList = webhookutil.EnsureBothFieldsAreEqual(r.Spec.DiskOffering.ID, r.Spec.DiskOffering.Name, oldSpec.DiskOffering.ID, oldSpec.DiskOffering.Name, "diskOffering", errorList)
```

Comment:
I'll remove this comment if I find the reason in my readthrough, but why would we require name and ID to be the same?

Does a disk offering not have a name and ID?